example/package.json

```diff
@@ -9,10 +9,12 @@
     "format": "npx prettier --write ."
   },
   "devDependencies": {
+    "@playwright/test": "^1.35.1",
     "pino-pretty": "^10.0.0",
     "prettier": "2.8.8"
   },
   "dependencies": {
+    "@pixelpact-hook/playwright-js": "file:../pixelpact-hooks/playwright-js",
```

Comment:
Let's discuss the naming of this again.
I'm not sure about how intuitive the word `hook` is. It's basically a client library - one per eco system (js, java, go, ...)
With that in mind, a simpler package name would be sufficient as there won't be any other npm packages, right?

Reply:
For example percy.io calls it: [`Playwright client library for visual testing with Percy`](https://github.com/percy/percy-playwright)